src/overlays/actors/ovl_En_Si/z_en_si.c

```diff
@@ -6,6 +6,8 @@
 
 #include "z_en_si.h"
 
+#define ENSI_GET_CHEST_FLAG(thisx) (((thisx)->params & 0xFC) >> 2)
```

Comment:
We prefer to put this in the header instead of the C file.

Reply:
Done! Thank you!